src/LendingPool.sol

```diff
@@ -1024,17 +1024,20 @@ contract LendingPool is LendingPoolGuardian, Creditor, DebtToken, ILendingPool {
 
             totalRealisedLiquidity = uint128(totalRealisedLiquidity - badDebt);
             _processDefault(badDebt);
+            (terminationReward, liquidationPenalty) = (0, 0);
```

Comment:
do we need this if they're already initialised at 0?

Reply:
![image](https://github.com/user-attachments/assets/2c0bf003-ba55-4f2e-a4c0-9c55757f43a7)
They are not?